src/main/java/opendcs/dao/CompDependsDAO.java

```diff
@@ -259,27 +241,38 @@ public void deleteCompDependsForTsKey(DbKey timeSeriesKey)
 		throws DbIoException
 	{
 		if (db.isHdb())
+		{
```

Comment:
What's different about the HDB instances that we're not doing deletions from the depends table?

Reply:
I have absolutely no idea which is why I left it in place.

Reply:
It's only called from a chain that deletes a Timeseries completely. We don't do that here.